src/renderer/services/network/chainsService.ts

```diff
@@ -45,21 +51,67 @@ export function useChains(): IChainService {
     let kusama;
     const testnets = [] as T[];
     const parachains = [] as T[];
+    const numberchains = [] as T[];
 
     chains.forEach((chain) => {
       if (isPolkadot(chain.name)) polkadot = chain;
       else if (isKusama(chain.name)) kusama = chain;
       else if (isTestnet(chain.options)) testnets.push(chain);
+      else if (isNameWithNumber(chain.name)) numberchains.push(chain);
       else parachains.push(chain);
     });
 
-    return compact([polkadot, kusama, ...sortBy(parachains, 'name'), ...sortBy(testnets, 'name')]) as T[];
+    return concat(
+      [polkadot, kusama].filter(nonNullable),
+      sortBy(parachains, 'name'),
+      sortBy(numberchains, 'name'),
+      sortBy(testnets, 'name'),
+    );
+  };
+
+  const sortChainsByBalance = (chains: Chain[], balances: Balance[]): Chain[] => {
```

Comment:
Isn't it better to change the name? Sorting works not only by balances.

Reply:
Good suggestion but I'm not sure about that
Name is pretty long already and if I change it to something like `sortChainsByGroupsNamesAndBalance` 
It would bring more questions to reader of the code at least about groups 
If he needs more details, he will jump into implementation anyway
So I would choose a compromise with current name 😊